2023/src/templates/speaker.tsx

```diff
@@ -112,19 +112,34 @@ export default function Speaker(props: Props) {
       />
       <ResponsiveBox>
         <Breadcrumb path={[{ label: t("speakers"), to: "/speakers" }, title]} />
-        {speakers.map((speaker, i) => (
-          <SpeakerBox key={speaker.uuid}>
-            <Avatar image={avatars[i]} alt={speaker.name} loading="eager" />
+        {speakers.map((speaker, i) => {
```

Comment:
In order for this to work, the parameters need to be added here: https://github.com/jsconfjp/jsconf.jp/blob/df31d3c26cecd8427eefff118b895e0644aa5e1f/2023/gatsby-node.js#L31-L35